polaris-react/src/components/Banner/components/BannerExperimental/BannerExperimental.tsx

```diff
@@ -137,3 +168,97 @@ export function BannerExperimental({
     </Box>
   );
 }
+
+function NoTitleBanner({
+  onDismiss,
+  backgroundColor,
+  bannerIcon,
+  actionButtons,
+  dismissButton,
+  children,
+}: PropsWithChildren<Omit<BannerLayoutProps, 'textColor' | 'bannerTitle'>>) {
+  const [blockAlign, setBlockAlgin] =
+    useState<HorizontalStackProps['blockAlign']>('center');
+  const contentNode = useRef<HTMLDivElement>(null);
+  const iconHeight = 28;
+
+  const handleResize = useCallback(() => {
+    if (
+      contentNode?.current?.offsetHeight &&
+      contentNode?.current?.offsetHeight > iconHeight
+    ) {
+      setBlockAlgin('start');
+    } else {
+      setBlockAlgin('center');
+    }
+  }, []);
+
+  useEffect(() => handleResize(), [handleResize]);
+
+  useEffect(() => {
+    if (!contentNode.current) return;
+    window.addEventListener('resize', handleResize);
+
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, [handleResize]);
+
+  return (
+    <Box width="100%" padding="3" borderRadius="2">
+      <HorizontalStack align="space-between" blockAlign="start" wrap={false}>
+        <Box paddingInlineEnd={onDismiss ? '2' : undefined}>
+          <HorizontalStack gap="2" wrap={false} blockAlign={blockAlign}>
+            <Box background={backgroundColor} borderRadius="2" padding="1">
+              {bannerIcon}
+            </Box>
+            <div ref={contentNode}>
+              <VerticalStack gap="2">
+                <div>{children}</div>
+                {actionButtons}
+              </VerticalStack>
+            </div>
+          </HorizontalStack>
+        </Box>
+        {dismissButton}
+      </HorizontalStack>
+    </Box>
+  );
+}
+
+function WithinContentContainerBanner({
```

Comment:
This code isn't new, I've just moved it from the ternary return type from the main function since there are 3 layouts now